### PR TITLE
Change character by frag identifier

### PR DESCRIPTION
**Pull request by cwade12c**
Completed **Todo: Use url parameters for girls instead of function calls**

- Added support to change the character by the fragment identifier. 
- Added a new function called loadWolf() to handle the switch statement. 
- window.onhashchange will pass the identifier to that function and window.onload will get the current location.hash and pass it to loadWolf. 
- Updated the index.html file to give each wolf a hash value.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,9 +21,9 @@
         <img src="res/wanRight.svg" onerror="this.src='res/wanRight.png'" class="wan_text" id="wanRight">
         <div id="wanCenter"></div>
         <div id="footer-info">
-            <a href="#" onclick="wolfMomiji()">Momiji</a>
-            <a href="#" onclick="wolfHolo()">Holo</a>
-            <a href="#" onclick="wolfKaren()">Karen</a>
+            <a href="#momiji" onclick="wolfMomiji()">Momiji</a>
```
Comment (cwade12c) [inline index.html:24]: Each a tag should have a hash value. This makes it easier to copy and paste a URL to send to others.

File: wolfscript.js [hunk 2]
```diff
@@ -130,6 +130,20 @@ function wolfKaren() {
     img.src = "girls/karen.png";
 }
 
+function loadWolf(character) {
```
Comment (cwade12c) [inline wolfscript.js:133]: This function is called by onload and onhashchange. We can probably reduce conditional complexity by introducing an MVC* pattern (as seen in AngularJS).

File: wolfscript.js [hunk 3]
```diff
@@ -158,6 +172,12 @@ window.onload = function () {
     box1.addEventListener("wekbkitAnimationEnd", boxAnimHandler, false);
     box1.addEventListener("MSAnimationEnd", boxAnimHandler, false);
     box1.addEventListener("animationend", boxAnimHandler, false);
+
+    loadWolf(location.hash);
+}
+
+window.onhashchange = function() {
```
Comment (cwade12c) [inline wolfscript.js:179]: Anytime that the fragment identifier updates, this function is called.

File: wolfscript.js [hunk 4]
```diff
@@ -158,6 +172,12 @@ window.onload = function () {
     box1.addEventListener("wekbkitAnimationEnd", boxAnimHandler, false);
     box1.addEventListener("MSAnimationEnd", boxAnimHandler, false);
     box1.addEventListener("animationend", boxAnimHandler, false);
+
+    loadWolf(location.hash);
```
Comment (cwade12c) [inline wolfscript.js:176]: Call loadWolf inside of the onload body since somebody might try to initially load the URL with a parameter in it.

**Comment by atomheartother** [comment]
Well this is horribly awkward, seems I [made this change](https://github.com/atomheartother/wanwan-html5/commit/8f216e7007e97a94888894a9a543058479d00465) and pushed it just as you opened this pull request... We pretty much did it the same way, though I also just completely removed the onClick events on the div elements since they're obsolete.

This is the weirdest thing, I'd love to merge your changes but we've pretty much done the same thing, thank you for contributing though!

**Pull request closed without merging**